clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/CTInAppWebView.java

```diff
@@ -47,7 +47,7 @@ public boolean performClick() {
     protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
         super.onMeasure(widthMeasureSpec, heightMeasureSpec);
         updateDimension();
-        setMeasuredDimension(dim.x, dim.y);
+        setMeasuredDimension(measureWidth(widthMeasureSpec,dim.x), measureHeight(heightMeasureSpec,dim.y));
```

Comment:
can you point out why this is changed?

Reply:
It's a bug when App is in edge-to-edge mode. Child must respect parent's required dimensions otherwise UI will break.